pkg/cmd/jsonpath.go

```diff
@@ -57,6 +52,45 @@ func (c jsonpathContext) jsonpath(
 	return nil
 }
 
+func evaluateJSONPath(path string, data interface{}, first bool) (interface{}, error) {
```

Comment:
This needs some commentary as to the necessity of the marshal/unmarshal/marshal sequence.

Reply:
Added.